CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 # Changelog
 
 <!-- next version -->
+### 💡 Enhancements 💡
+- `contrib`: Add the component loki exporter to the distribution (#892)
```

Comment:
Please don't edit changelog.md directly, instead add an entry in .chloggen

Reply:
Oh, missed it. Fixed 👍